llvm-external-projects/iree-dialects/lib/Dialect/VectorExt/IR/VectorExtAttrs.cpp

```diff
@@ -491,129 +389,45 @@ LogicalResult NestedLayoutAttr::verify(
 /// threads -> elements). There is no requirement that a subgroup id only
 /// spans subgroups.
 SmallVector<Value>
-NestedLayoutAttr::computeThreadIds(Value threadId,
+NestedLayoutAttr::computeThreadIds(Value threadId, int64_t subgroupSize,
                                    RewriterBase &rewriter) const {
-  // The subgroup/thread bases tell us the ranges of the corresponding id from
-  // slowest varying to fastest varying. Thus to get the correct basis ids, we
```

Comment:
Would be nice if can we leave some high-level/intuition comment/doc as seen here?